htdocs/lib/MapMarkers.js

```diff
@@ -0,0 +1,680 @@
+        div.style['background-size'] = '384px 144px';
+        div.style['background-position-x'] = -(this.symbol.index % 16) * 24 + 'px';
+        div.style['background-position-y'] = -Math.floor(this.symbol.index / 16) * 24 + 'px';
+    }
+
+    // If entity is flying at a significant altitude...
+    if (this.altitude >= 500) {
+        // r = elevation, a = rotation, <x,y> = shadow offset
+        var r = Math.round(this.altitude / 1000);
+        var a = - Math.PI * (this.course? this.course - 45 : -45) / 180;
+        var x = r * Math.cos(a);
+        var y = r * Math.sin(a);
+        div.style.filter = 'drop-shadow(' + x + 'px ' + y + 'px 0.5px rgba(0,0,0,0.5))';
+    } else {
+        div.style.filter = 'none';
+    }
+
+    if (!this.course) {
+        div.style.transform = null;
+    } else if (this.symbol && !this.isFacingEast(this.symbol)) {
+        // Airplanes and other symbols point up (to the north)
+        div.style.transform = 'rotate(' + this.course + 'deg)';
+    } else if (this.course > 180) {
+        // Vehicles and vessels point right (to the east)
+        div.style.transform = 'scalex(-1) rotate(' + (270 - this.course) + 'deg)'
+    } else {
+        // Vehicles and vessels point right (to the east)
+        div.style.transform = 'rotate(' + (this.course - 90) + 'deg)';
+    }
+
+    if (this.symbol && this.symbol.table !== '/' && this.symbol.table !== '\\') {
+        overlay.style.display = 'block';
+        overlay.style['background-position-x'] = -(this.symbol.tableindex % 16) * 24 + 'px';
+        overlay.style['background-position-y'] = -Math.floor(this.symbol.tableindex / 16) * 24 + 'px';
+    } else {
+        overlay.style.display = 'none';
+    }
+
+    if (this.opacity) {
+        div.style.opacity = this.opacity;
+    } else {
+        div.style.opacity = null;
+    }
+
+    if (this.place) this.place();
+};
+
+AprsMarker.prototype.create = function() {
+    var div = this.div = document.createElement('div');
+
+    div.style.position = 'absolute';
+    div.style.cursor = 'pointer';
+    div.style.width = '24px';
+    div.style.height = '24px';
+
+    var overlay = this.overlay = document.createElement('div');
+    overlay.style.width = '24px';
+    overlay.style.height = '24px';
+    overlay.style.background = 'url(aprs-symbols/aprs-symbols-24-2@2x.png)';
+    overlay.style['background-size'] = '384px 144px';
+    overlay.style.display = 'none';
+
+    div.appendChild(overlay);
+
+    return div;
+};
+
+AprsMarker.prototype.getAnchorOffset = function() {
+    return [0, -12];
+};
+
+AprsMarker.prototype.getInfoHTML = function(name, receiverMarker = null) {
+    var timeString = moment(this.lastseen).fromNow();
+    var commentString = '';
+    var weatherString = '';
+    var detailsString = '';
+    var hopsString = '';
+    var distance = '';
+
+    if (this.comment) {
+        commentString += '<p>' + Marker.makeListTitle('Comment') + '<div>' +
```

Comment:
HTML does not allow the use of `<div>` inside `<p>`. This results in some strange layout issues.

Reply:
Oh, really? That is new to me. Is there a reference to this quirk somewhere?

Reply:
From what I understand, this is part of the HTML specification. The first answer on [this stackoverflow question](https://stackoverflow.com/questions/10763780/putting-div-inside-p-is-adding-an-extra-p) explains it better than I could do, and links to the relating parts of the spec.

I have fallen victim to this myself. I only stumbled across this specific case here while I was trying to figure out how the content of APRS markers is rendered.

You can also see that the resulting DOM tree does not correspond to the markup that is generated here if you look at it in the inspector.

Reply:
Holy... Anyway, I have fixed these in OWRX+. Thanks for pointing them out. @0xAF this is the latest MapMarkers.js with the fixes:
https://github.com/luarvique/openwebrx/blob/master/htdocs/lib/MapMarkers.js

Reply:
ported to this PR in the latest commits.